query-connector/src/styles/custom-styles.scss

```
@@ -451,38 +451,41 @@ hr.custom-hr {
   width: calc($tefca-viewer-max-width * 0.4) !important;
 }
 
-.customize-accordion .usa-accordion__button[aria-expanded="true"],
-.customize-accordion .usa-accordion__button[aria-expanded="false"] {
-  background-size: 16px 10px;
-  background-position: top 24px right 20px;
-}
+.custom-query-step-indicator .usa-step-indicator__header {
+  display: none !important;
+  .customize-accordion .usa-accordion__button[aria-expanded="true"],
+  .customize-accordion .usa-accordion__button[aria-expanded="false"] {
+    background-size: 16px 10px;
+    background-position: top 24px right 20px;
+  }
```

also I just realized: did we mean to nest all the classes below here into the header class? The indentation here is saying that these classes are now subclasses of the header styling we changed here, which might not be what we want

```suggestion
.custom-query-step-indicator .usa-step-indicator__header {
  display: none !important;
 }
.customize-accordion .usa-accordion__button[aria-expanded="true"],
.customize-accordion .usa-accordion__button[aria-expanded="false"] {
  background-size: 16px 10px;
  background-position: top 24px right 20px;
}
```